sbi/inference/snpe/snpe_base.py

```diff
@@ -269,6 +227,18 @@ def _run_simulations(
 
         return theta, x, self._mask_sims_from_prior(round_, theta.size(0))
 
+    def _z_score_embedding(self, x: Tensor) -> nn.Module:
+        """Return embedding net with a standardizing step preprended."""
+
+        # XXX Mouthful, rename self.posterior.nn
+        embed_nn = self._neural_posterior.neural_net._embedding_net
```

Comment:
Yes, let's rename this!

Reply:
(maybe some other time when there are not 5 PRs open though ;) )

Reply:
Of course, the comment is there precisely to defer such refactorings.